cms/cache/page.py

```diff
@@ -45,30 +45,37 @@ def set_page_cache(response):
 
     placeholders = toolbar.content_renderer.get_rendered_placeholders()
     # Checks if there's a plugin using the legacy "cache = False"
-    placeholder_ttl_list = []
+    ttl_list = []
     vary_cache_on_set = set()
     for ph in placeholders:
         # get_cache_expiration() always returns:
         #     EXPIRE_NOW <= int <= MAX_EXPIRATION_IN_SECONDS
         ttl = ph.get_cache_expiration(request, timestamp)
         vary_cache_on = ph.get_vary_cache_on(request)
 
-        placeholder_ttl_list.append(ttl)
+        ttl_list.append(ttl)
         if ttl and vary_cache_on:
             # We're only interested in vary headers if they come from
             # a cache-able placeholder.
             vary_cache_on_set |= set(vary_cache_on)
 
-    if EXPIRE_NOW not in placeholder_ttl_list:
-        if placeholder_ttl_list:
-            min_placeholder_ttl = min(x for x in placeholder_ttl_list)
-        else:
-            # Should only happen when there are no placeholders at all
-            min_placeholder_ttl = MAX_EXPIRATION_TTL
-        ttl = min(
-            get_cms_setting('CACHE_DURATIONS')['content'],
-            min_placeholder_ttl
-        )
+    if EXPIRE_NOW not in ttl_list:
+        ttl_list.append(get_cms_setting('CACHE_DURATIONS')['content'])
+        ttl_list.append(MAX_EXPIRATION_TTL)
+
+        # If defined the `CMS_CACHE_LIMIT_TTL_MODULE` extension point
+        if hasattr(settings, 'CMS_CACHE_LIMIT_TTL_MODULE'):
```

Comment:
From what I can see, `CMS_CACHE_LIMIT_TTL_MODULE` is new in this PR.

Therefore we'll need some documentation to show what this is expected to be, and how to use it.

I think some detail in here; https://docs.django-cms.org/en/latest/how_to/caching.html?highlight=cache

And also something more brief in the advanced settings; https://docs.django-cms.org/en/latest/reference/configuration.html?highlight=settings#advanced-settings

Reply:
I've included the requested documentation.